### ISSUE TITLE
Question re bugfixes

### ISSUE THREAD

**Issue by Syst3mSh0ck** [phobos2077/sfall#205]
Hi. Thanks for the great work so far. Could I compile the Sfall from the latest commit in git then use the .dll as a drop-in replacement for existing mods such as MIB or Restoration Mod (providing I backup the INI and resolve the diffrences afterwards)? The reason I ask is because I thought you may have resolved some bugs in the function calls / hooks already used by those mods? Cheers. :)

**Comment by NovaRain** [comment]
Yes, you can use your compiled .dll to replace the one in other mods as long as you've taken care of the INI,
